test/e2e/discovery/service_discovery.go

```diff
@@ -97,7 +97,7 @@ func verifyServiceIpWithDig(f *framework.Framework, cluster framework.ClusterInd
 		serviceIP = service.Spec.ClusterIP
 	}
 
-	cmd := []string{"dig", service.Name + "." + f.Namespace + ".svc.cluster" + strconv.Itoa(int(cluster+1)) + ".local", "+short"}
+	cmd := []string{"dig", service.Name + "." + f.Namespace + ".svc.supercluster" + strconv.Itoa(int(cluster+1)) + ".local", "+short"}
```

Comment:
```suggestion
	cmd := []string{"dig", service.Name + "." + f.Namespace + ".svc.supercluster.local", "+short"}
```

Reply:
But you also need to remove strconv, or it will err at compile

Reply:
yeah, done that, probably while you were commenting this :D